pkg/api/validation/schema.go

```diff
@@ -24,7 +24,7 @@ import (
 
 	"github.com/emicklei/go-restful/swagger"
 	"github.com/golang/glog"
-	"gopkg.in/v2/yaml"
+	"gopkg.in/yaml.v2"
```

Comment:
Not me, sorry.

On Tue, Jan 13, 2015 at 4:12 PM, Daniel Smith <notifications@github.com>
wrote:

> In pkg/api/validation/schema.go
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/3440#discussion-diff-22907289>
> :
>
> > @@ -24,7 +24,7 @@ import (
> >
> >  	"github.com/emicklei/go-restful/swagger"
> >  	"github.com/golang/glog"
> > -	"gopkg.in/v2/yaml"
> > +	"gopkg.in/yaml.v2"
>
> @thockin <https://github.com/thockin> I seem to recall you wanting this
> change in a few places but I can't recall why?
>
> --
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/3440/files#r22907289>
> .
>